### ISSUE TITLE
controls.dryRun and controls.traceResponse breaking GCP service discovery

### ISSUE THREAD

**Issue by jfreeland** [F5Networks/f5-appsvcs-extension#493]
### Environment
 * Application Services Version: 3.30.0-5
 * BIG-IP Version: 15.1.3

### Summary
https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/as3-api.html#query-parameters-for-controls-objects were released in this version of AS3.

When I try to apply an AS3 declaration with ansible to the relevant endpoint, service discovery stops working for me; my nodes go become unhealthy and my virtual servers go down.

```
- name: Diff AS3 from node
  uri:
    url: https://{{ inventory_hostname }}:{{ provider.server_port }}/mgmt/shared/appsvcs/declare?controls.dryRun=true&controls.traceResponse=true
    method: PATCH
    return_content: yes
    validate_certs: false
    user: "{{ F5_USER }}"
    password: "{{ F5_PASSWORD }}"
    status_code: [200]
    body: "{{ lookup('file', new_as3) | from_json }}"
    body_format: json
  delegate_to: localhost
  register: as3_diff
```

It's unrelated to the issue I'm experiencing, but I'd like to be able to use https://github.com/f5devcentral/ansible-role-f5_atc_deploy_declaration/issues/42 for this.

### Steps To Reproduce
1. Submit any declaration, whether there are changes or not, using the controls objects to declare a dryRun with traceResponse to a node that uses GCP service discovery.

2. Observe the state of your pool members.

### Expected Behavior
No changes to the operation of the node.  This should be a dryRun.

### Actual Behavior
My backend pool members using GCP service discovery go down.

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this to our internal product backlog as AUTOTOOL-2690.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.32.0 release.
